contracts/crypto-verify/src/contract.rs

```diff
@@ -0,0 +1,192 @@
+use cosmwasm_std::{
+    attr, entry_point, to_binary, Binary, Deps, DepsMut, Env, MessageInfo, QueryResponse, Response,
+    StdResult,
+};
+
+use crate::msg::list_verifications;
+use crate::msg::{HandleMsg, InitMsg, ListVerificationsResponse, QueryMsg};
+
+pub const VERSION: &str = "crypto-verify-v1";
+
+#[entry_point]
+pub fn init(_deps: DepsMut, _env: Env, _info: MessageInfo, _msg: InitMsg) -> StdResult<Response> {
+    // Does nothing
+    Ok(Response {
+        messages: vec![],
+        attributes: vec![attr("action", "init")],
+        data: None,
+    })
```

Comment:
Let's remove any unnecessary noise from here

```suggestion
    Ok(Response::default())
```